src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,79 @@
+package edu.kis.vh.nursery;
+
+/**
+ * Klasa DefaultCountingOutRhymer zapewnia prostą implementację stosu
+ * o stałej pojemności, używaną do demonstrowania podstawowych operacji na strukturach danych.
+ */
+public class DefaultCountingOutRhymer {
+
+    /**
+     * Stała reprezentująca stan pusty.
+     */
+    public static final int EMPTY = -1;
+
+    /**
+     * Stała reprezentująca pojemność stosu.
+     */
+    public static final int CAPACITY = 12;
+
+    /**
+     * Tablica do przechowywania elementów stosu.
+     */
+    private final int[] numbers = new int[CAPACITY];
+
+    /**
+     * Indeks ostatniego elementu na stosie, lub EMPTY, jeśli stos jest pusty.
+     */
+    private int total = EMPTY;
+
+    /**
+     * Dodaje element do stosu, jeśli nie jest pełny.
+     *
+     * @param in element do dodania do stosu
+     */
+    public void countIn(int in) {
+        if (!isFull())
+            numbers[++total] = in;
+    }
+
+    /**
+     * Sprawdza, czy stos jest pusty.
+     *
+     * @return true, jeśli stos jest pusty, w przeciwnym razie false
+     */
+    public boolean callCheck() {
+        return total == EMPTY;
```

Comment:
Errors in collaborators work (lack of good review). Extracting constants is done incorrectly. Have you used review hints ('Github - PR checkpoint' at wikamp).
This constant should be different (separate) than the one in peekaboo and countOut methods.